website/docs/admin_hosts_maya.md

```diff
@@ -110,40 +106,28 @@ or Deadlines **Draft Tile Assembler**.
 This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
 `Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.
 
-### Extract Playblast Settings (review)
-These settings provide granular control over how the playblasts or reviews are produced in Maya.
-
-Some of these settings are also available on the instance itself, in which case these settings will become the default value when creating the review instance.
-
-![Extract Playblast Settings](assets/maya-admin_extract_playblast_settings.png)
-
-- **Compression type** which file encoding to use.
-- **Data format** what format is the file encoding.
-- **Quality** lets you control the compression value for the output. Results can vary depending on the compression you selected. Quality values can range from 0 to 100, with a default value of 95.
-- **Background Color** the viewports background color.
-- **Background Bottom** the viewports background bottom color.
-- **Background Top** the viewports background top color.
-- **Override display options** override the viewports display options to use what is set in the settings.
-- **Isolate view** isolates the view to what is in the review instance. If only a camera is present in the review instance, all nodes are displayed in view.
-- **Off Screen** records the playblast hidden from the user.
-- **2D Pan/Zoom** enables the 2D Pan/Zoom functionality of the camera.
-- **Renderer name** which renderer to use for playblasting.
-- **Width** width of the output resolution. If this value is `0`, the asset's width is used.
-- **Height** height of the output resolution. If this value is `0`, the asset's height is used.
-
-#### Viewport Options
```

Comment:
This looks like you accidentally deleted some settings on a merge conflict. I suspect these shouldn't be removed. Right? There some other changes in this file too that I suspect shouldn't be here either.

Reply:
Oh yeah I just realized I removed some things that I shouldn't have ! I will revert my changes